Makefile

```
@@ -59,3 +59,21 @@ yarn-install-deps: $(UI_INSTALL_ARTIFACT)
 .PHONY: package
 package: yarn-install-deps
 	cd ui && yarn build && mkdir -p ../target && zip -r ../target/bondui.zip build/
+
+# DABL specific
+deploy: clean build yarn-install-deps
```

```suggestion
deploy: package yarn-install-deps
```
(please do not add `clean` as a dep)

---

why so?

---

why do you need it? it _should_ work without it.
it suggests that we don't believe that the dependency handling of the Makefile works. it can also cause troubles with parallel execution (`make -j`).
if you want to force rebuild, you can still use `make -B`, but that's also an admittance of a failing Makefile.